pytorch_lightning/trainer/connectors/accelerator_connector.py

```diff
@@ -248,6 +254,10 @@ def on_cpu(self) -> bool:
     def on_tpu(self) -> bool:
         return self.tpu_cores is not None
 
+    @property
+    def on_ipu(self) -> bool:
+        return self.ipus is not None
```

Comment:
Maybe return 0 here, if IPU is not available.

Reply:
Not sure if this is necessary, as like tpus if a user specifies ipus and they are not available, it should crash